pkg/prometheus/common.go

```diff
@@ -439,3 +446,20 @@ func BuildWebconfig(
 
 	return webConfig.GetMountParameters()
 }
+
+func scrapeFailureLogFileVolume(file string) (v1.Volume, v1.VolumeMount, bool) {
+	if !UsesDefaultFileVolume(file) {
+		return v1.Volume{}, v1.VolumeMount{}, false
+	}
+
+	return v1.Volume{
```

Comment:
if both query and scrape failure log files are configured, it will mount 2 emptyDir volumes at the same mountpath which isn't ideal. It's something that we could check in appendServerVolumes().

Reply:
Thanks @simonpasquier .
Fixed.
Please re-check it